### ISSUE TITLE
Pagination of solutions (and likely other things) doesn't move focus when page changes

### ISSUE THREAD

**Issue by SleeplessByte** [exercism/exercism#6514]
Linking [Accessible Pagination](https://www.a11ymatters.com/pattern/pagination/). Note that basically everything from that document _has_ been implemented. A few comments of what's currently implemented and what I expected.

- Pagination is using `button` but those buttons are without type. This is fine because they are not contained in a form, but explicitly labelling them as `button`, would be good because it's indicative that it has userland defined behaviour.
- ...but pagination changes the URL. This means that those buttons should really be links.
- ...because the URL changes, the content changes, and that new content isn't really announced to the user. 
- ...you cannot put `rel="next"` on buttons, but you probably want to.

I recommend two things:

1. Update the title to say page x out of y (e.g. Page 3 out 7 of SleeplessByte JavaScript Solutions)
2. Move the focus to the paginated content

There's probably more work to be done.
